booster/tools/booster/Server.hs

```diff
@@ -154,39 +157,51 @@ main = do
 
             monadLogger <- askLoggerIO
 
-            koreLogEntriesAsJsonSelector <-
-                if Logger.LevelOther "SimplifyJson" `elem` customLevels
-                    then case Map.lookup (Logger.LevelOther "SimplifyJson") logLevelToKoreLogEntryMap of
-                        Nothing -> do
-                            Logger.logWarnNS
-                                "proxy"
-                                "Could not find out which Kore log entries correspond to the SimplifyJson level"
-                            pure (const False)
-                        Just koreSimplificationLogEntries -> pure (`elem` koreSimplificationLogEntries)
-                    else pure (const False)
+            let koreLogRenderer = case logFormat of
+                    Standard -> renderStandardPretty (ExeName "") (TimeSpec 0 0) TimestampsDisable
+                    OneLine -> renderOnelinePretty (ExeName "") (TimeSpec 0 0) TimestampsDisable
+                    Json -> renderJson (ExeName "") (TimeSpec 0 0) TimestampsDisable
+                koreLogLateFilter = case logFormat of
+                    OneLine ->
+                        if contexLoggingEnabled
+                            then \txt ->
+                                -- FIXME: likely terrible performance! Use something that does not unpack Text
+                                not (any (flip Glob.match (Text.unpack txt)) negGlobPatterns)
+                                    && any (flip Glob.match (Text.unpack txt)) globPatterns
```

Comment:
We could shorten the string to only match the context prefix.
Unfortunately `regex-pcre` - which we already use - does not support `Text`, but a more crude method could be used: use `Text.takeWhile` to cut the log text at the first character that is known to not be in the prefix. Characters in the prefix, IIUC, are hex digits, small letter (by convention!), space, and opening/closing angle brackets.' Or we could add a known separator character (like `:`) after the context prefix.
Maybe it will help...

Reply:
Good idea, I've implemented that.